packages/stryker-mutator-specification/src/ObjectMutatorSpec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ObjectMutatorSpec(
+  name: string,
+  expectMutation: ExpectMutation
+) {
+  describe('ObjectMutator', () => {
+    it('should have name "Object"', () => {
```

Comment:
`"Object"` mutator sounds pretty generic. I think `ObjectLiteral` would be a better fit.

Reply:
Could we also add a test for:

```js
{ ['key']: 'value' } // mutated to { }
```

What do you think?

Reply:
@nicojs good point on the name - have amended both points